bin/cron/pardot_sync

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env ruby
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require_relative '../../lib/state_abbr'
+require src_dir 'database'
+require_relative('../../dashboard/config/environment')
+require 'cdo/properties'
+require 'json'
+
+DASHBOARD_REPORTING_DB_READONLY = sequel_connect(
+  CDO.dashboard_reporting_db_reader, CDO.dashboard_reporting_db_reader)
+
+UPDATE_BATCH_SIZE = 100
+
+DASHBOARD_DB_NAME = "dashboard_#{Rails.env}"
+
+def log(s)
+  puts s      # emit to stdout
+  CDO.log.info s   # emit to log file
+end
+
+def truncate_contacts_rollup
+  start = Time.now
+  log "Truncating contacts_rollup"
+  PEGASUS_DB.run "TRUNCATE TABLE contacts_rollup"
+  log "   Completed in " + ((Time.now) - start).to_s + " seconds"
+end
+
+def insert_from_dashboard_contacts
+  start = Time.now
+  log "Inserting teacher contacts and IP geo data from dashboard.users"
+  PEGASUS_DB.run "
+    INSERT INTO contacts_rollup (email,name,dashboard_user_id,roles, city, state, postal_code, country)
+	  SELECT email COLLATE utf8_general_ci, name, users.id, 'Teacher', city, state, postal_code, country FROM #{DASHBOARD_DB_NAME}.users AS users
```

Comment:
Given that all our codebase roles are lowercase (e.g., `User::TYPE_TEACHER` is `'teacher'`, `UserPermission::FACILITATOR` is `facilitator`, etc.), for consistency, can `roles` be a comma-separated list of lowercase_underscore_separated_strings?